modules/react/icon/lib/Graphic.tsx

```diff
@@ -26,11 +25,39 @@ export interface GraphicStyles {
   grow?: boolean;
 }
 
-export interface GraphicProps extends GraphicStyles, Pick<SvgProps, 'shouldMirror' | 'cs'> {
+export interface GraphicProps extends CSProps {
   /**
-   * The graphic to display from `@workday/canvas-graphics-web`.
+   * The height of the graphic
    */
-  src: CanvasGraphic;
+  height?: number | string;
+  /**
+   * The width of the graphic
+   */
+  width?: number | string;
+  /**
+   * If true, expand the Graphic to fit its container. `grow` takes precedence over both `width` and `height`.
+   */
+  grow?: boolean;
+  /**
+   * The source of the graphic. If `svg` is provided, it will create a Base64-encoded ASCII string from a binary string (i.e., a string in which each character in the string is treated as a byte of binary data) and pass it to an image `src`. If `url` is provided, it will be rendered as an image via the `src` property..
```

Comment:
```suggestion
   * The source of the graphic. If `svg` is provided, it will create a Base64-encoded ASCII string from a binary string (i.e., a string in which each character in the string is treated as a byte of binary data) and pass it to an image `src`. If `url` is provided, it will be rendered as an image via the `src` property.
```